### DISCUSSION TITLE
Thumbs up/down errors, playback on iPhone problems and more.

### DISCUSSION THREAD

**Discussion by AnthonyTorreso** [mediacms-io/mediacms#999] [Q&A]
First off this is an amazing piece of software that I love, and for my needs is exactly what I am looking for. 
Some unique things I have found with this.

Setup:
 Debian 12
 1GB RAM
 MediaCMS 3.1.0
 Standalone install, no docker.

Demo: https://videos.theserveradmin.com/
Uploads/Register Disabled
No encoding, stream original
Downloaded some YouTube videos for a demo, so no need to encode.

Some items that I noted, not any sort of deal breakers, but some odd things none the less.

Tried on:

    Chrome on Windows
    Firefox on Windows
    Edge on Wunbuntu
    Firefox on Mint
    Brave on Mint
    Opera on Android
    Safari and Chrome on iPhone

What worked on all devices/OSes:
- Full playthrough
- Skipping forward and back
- 1.25x, 1.5x, 1.75x, 2x
- Autoplay to next video
- All buttons and controls
- 5-7 concurrent sessions running

What did not work as expected:

    Number of views seems incorrect. I spun up Tomorrowland at least 7x and played it all full through numerous times, yet it shows only 6 views. I am thinking it should be over 10.
    Not all Thumbs up worked. I hit thumbs up on Tomorrowland 7x (on all of the above) and only 3 show when I refresh. I used 7 different browsers, 4 OSs, 4 devices. Unless it goes by router IP address -- but I found that weird.  If me and my wife see it and thumbs it up, it should be counted as 2.
    The Thumbs up and down on Opera/Android changes finger to green but does not show numbers. Same on iPhone. It should show the counts.
    On "Monkey Man" I did a thumbs down and it resulted in an "Action Failed" error (this was Firefox on Mint). However, the thumbs down stuck. Just a rogue error, benign, no biggie.
        Tried again on the following and it resulted in the error:
        Tales of the Empire (Edge on Wubuntu)
        Godzilla & Kong Vs Shimo & Skar King - Fight Scene (Firefox on Windows)
        So I think there is some internal code failure with that option ?
    iPhone does not recognize this site as a video site and it shuts off after 5 minutes of viewing. Android recognizes it and does not shut off. On iPhone, I tried with Safari and Chrome, and both times, phone blacked out at 5 minutes, same as if I left it unattended.

Hope the feedback helps...

Thanks so much,
Anthony

**Comment by jackson-xp** [comment]
And  When you click thumbs up or down, they will both turn green simultaneously.  Your website runs very smoothly, wow! Is it running on 1GB of memory and a single CPU core? Is it SSD or HDD?

**Comment by AnthonyTorreso** [comment]
Good Morning, 
  Yes this is a Linode Nano, so 1 GB Ram, 1 Shared vCPU, and SSD.  Like I said, this MediaCMS is truly amazing, and for the applications that I have some plans for, the problems I found above can all be disabled, but my business partner is always hell bent on finding "bugs" or bumps in the road that any customer might find.
Thank you.
Anthony

**Reply by jackson-xp** [reply]
have you tried this one，seeems pretty cool，far more features than mediacms

https://github.com/mediacms-io/mediacms
